api/app/core/matrix_validation.py

```
@@ -0,0 +1,146 @@
+
+def validate_message(message: str) -> None:
+    """Validate message content.
+
+    Args:
+        message: The message content to validate
+
+    Raises:
+        ValidationError: If message fails validation
+    """
+    if not isinstance(message, str):
+        raise ValidationError(f"Message must be a string, got {type(message)}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)